### PR TITLE
Switch security techfab to produce magazines

### PR DESCRIPTION
**Pull request by Partmedia**
## About the PR
This removes individual cartridge lathe recipes from the security techfab in favor of new recipes for all of the magazines in common use by NanoTrasen security.

## Why / Balance
- [People don't like individual cartridges](https://discord.com/channels/310555209753690112/675078881425752124/1147710750912151593)
- The emagged autolathe is untouched. Syndicate agents can continue to painstakingly fill their magazines manually.
- The special (rubber/incindiary) rounds are not touched

## Media
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: notafet
- tweak: Security techfabs now produce magazines instead of individual cartridges.

### CODE REVIEW

**Comment by Aisu9** [comment]
the detective could use the magnum ammo...

**Comment by LankLTE** [comment]
magnum rounds are for the detective.
I'd say keep the ammo boxes (in a ddition to the mags) since in my experience they're still useful, and without them you have no way to refill current magazines which may lead to a bunch of empty mags lying around and also just feels weird

**Comment by Partmedia** [comment]
Oversight. Sure, I'll restore all the ammo boxes.

**Comment by Aisu9** [comment]
may i suggest also adding a fully loaded speedloader crafting recipe for the detective's revolver?

**Comment by deltanedas** [comment]
boxes giving you carpal tunnel could be solved by adding a doafter to fill a mag from the box/other mag rather than spam clicking

**Comment by Partmedia** [comment]
> may i suggest also adding a fully loaded speedloader crafting recipe for the detective's revolver?

I thought about this, but somehow I remember speedloaders as intended to be syndicate-only, but since I can't find the relevant discussion and multiple people asked for this on Discord, I'm inclined to add them.

**Comment by Partmedia** [comment]
> boxes giving you carpal tunnel could be solved by adding a doafter to fill a mag from the box/other mag rather than spam clicking

I welcome you to work on this code.

**Comment by deltanedas** [comment]
bet

**Comment by Aisu9** [comment]
> I thought about this, but somehow I remember speedloaders as intended to be syndicate-only, but since I can't find the relevant discussion and multiple people asked for this on Discord, I'm inclined to add them.

i mean... the "syndicate" speedloader have syndicate calibre magnum bullets and not standard magnum loaded in

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 1]
```diff
@@ -518,13 +518,14 @@
       - ForensicPad
       - ClothingEyesGlassesSecurity
       - RiotShield
-      - CartridgePistol
-      - CartridgeMagnum
       - ShellShotgun
       - ShellShotgunFlare
       - ShellTranquilizer
-      - CartridgeLightRifle
-      - CartridgeRifle
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:527]: i think some of these LatheRecipePrototype are never used again, remove em as well if so, also figure out what to do with magnum as its a pretty common weapon

**Review by AJCM-git — DISMISSED** [review]

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml (hunk 1 above)
Comment (Partmedia) [reply Resources/Prototypes/Entities/Structures/Machines/lathe.yml:527]: No, they're used in the emagged autolathe, the ammo printer (that is not mapped), and presumably some forks and so deserve to stick around.

I readded the magnum ammo box.

**Comment by deltanedas** [comment]
i dont see why ammo boxes should be removed, make a few mags for your gun then a few extra boxes for a twice the ammo in half the storage space, excellent for salvage or anyone else that needs long term ammo

**Comment by LankLTE** [comment]
The ammo boxes were already added back

**Comment by deltanedas** [comment]
o

**Comment by Aisu9** [comment]
So Partmedia, speedloaders are not syndicate technology
xdgdg

**Comment by LankLTE** [comment]
> I thought about this, but somehow I remember speedloaders as intended to be syndicate-only, but **since I can't find the relevant discussion and multiple people asked for this on Discord, I'm inclined to add them.**

You saw this reply, right? I believe they just never got to it. It'd be a simple PR and I don't personally have any issues with it.

**Comment by Partmedia** [comment]
Sure, we can add speedloaders then.

**Pull request merged**